mlxtend/plotting/plot_sequential_feature_selection.py

```diff
@@ -10,6 +10,7 @@
 
 
 def plot_sequential_feature_selection(metric_dict,
+                                      figsize=None
```

Comment:
I think you are missing a comma here after "None"

Reply:
Fixed it